src/app/controllers/CartController.js

```diff
@@ -0,0 +1,97 @@
+class CartController {
+    constructor(app) {
+        this.app = app;
+        // ADD BINDINGS BELOW
+        this.removeItem = this.removeItem.bind(this);
+        this.addItem = this.addItem.bind(this);
+        this.create = this.create.bind(this);
+    }
+    
+    // ADD MUTATION METHODS BELOW
+
+    /**
+     * Remove an item from currently active cart.
+     * @param cartId
+     * @param cartItemId
+     * @returns {Promise<any>}
+     */
+    removeItem(cartId, cartItemId){
+        return new Promise((resolve, reject) => {
+            let mutationString = `
+                mutation ($cartId: String!, $cartItemId: String!) {
+                    removeItemCart(
+                        cart_id: $cartId,
+                        cart_item_id: $cartItemId
+                    ) {
+                        _id
+                    }
+                }
+            `;
+            this.app.getAdaptor().mutate(mutationString, {
+                cartId, cartItemId
+            }).then((result) => {
+                resolve(result.removeItemCart);
+            }).catch(e => {
+                reject(e);
+            });
+        });
+    }
+
+    /**
+     * Add an new item to currently active cart.
+     * @param cartId
+     * @param item
+     * @returns {Promise<any>}
+     */
+    addItem(cartId, item){
+        return new Promise((resolve, reject) => {
+            let mutationString = `
+                mutation ($cartId: String!, item: AddItemToCartInput!) {
```

Comment:
Something to think about, I believe these could be extracted into a fragment or separated into another file so that there is no need to bulk up main code base with useless queries lol